bob-common/src/mapper.rs

```diff
@@ -116,24 +122,26 @@ impl Virtual {
     }
 
     pub fn get_support_nodes(&self, key: BobKey, count: usize) -> Vec<&Node> {
```

Comment:
Please, add shortcut for situation when `count == 0`:
```
if count == 0 {
    return vec![];
}

Reply:
Done.